### DISCUSSION TITLE
DrawIO is definitely one of the best wireframe builders

### DISCUSSION THREAD

**Discussion by jdrake6789** [jgraph/drawio#2131] [General]
I just wanted to say that DrawIO is absolutely badass. It's one of the best wireframe builders I've used. Way better than Adobe XD. A huge thank you for making this amazing open source software.

**Comment by Bastesque** [comment]
@jdrake6789 That sounds pretty interesting. I am new to the principle of wireframes but I am willing to learn. Would you mind sharing the facts that lead to your impression so I/we can profit from it? No need for long text, a few bullets are enough for me to try it out myself. Thank you ;)
